ldcbench.data-generator/src/main/java/org/dice_research/ldcbench/benchmark/DataGenerator.java

```diff
@@ -225,6 +234,47 @@ protected void addInterlinks(GraphBuilder g) {
         }
     }
 
+    /**
+     * Append a given number of Blank Nodes to an existing graph.
+     * Edges are created to link to the Blank Nodes.
+     *
+     * @param g the graph
+     * @param nodeCount
+     *            the number of blank nodes
+     * @param seed
+     */
+    private void addBlankNodes(GraphBuilder g, int nodeCount, long seed) {
+        int blankNodesIndex = g.getNumberOfNodes();
+        g.setBlankNodesRange(blankNodesIndex, nodeCount);
+        addBNodesOrLiterals(g, blankNodesIndex, nodeCount, seed);
+    }
+
+    /**
+     * Append a given number of literals to an existing graph.
+     * Edges are created to link to the literals.
+     *
+     * @param g the graph
+     * @param nodeCount
+     *            the number of literals
+     * @param seed
+     */
+    private void addLiterals(GraphBuilder g, int nodeCount, long seed) {
+        int literalIndex = g.getNumberOfNodes();
+        g.setLiteralsRange(literalIndex, nodeCount);
+        addBNodesOrLiterals(g, literalIndex, nodeCount, seed);
+    }
+
+    private void addBNodesOrLiterals(GraphBuilder g, int index, int NumberOfNodes, long seed) {
```

Comment:
`NumberOfNodes` is not a valid parameter name since it starts with a capital letter.

Reply:
This issue is still not fixed.